documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -117,9 +138,9 @@ You can migrate virtual machines (VMs) by using the {ocp} web console to:
 ====
 You must ensure that all xref:prerequisites_{context}[prerequisites] are met.
 
-VMware only: You must have the minimal set of xref:vmware-privileges_{context}[VMware privileges].
+// VMware only: You must have the minimal set of xref:vmware-privileges[VMware privileges].
```

Comment:
Is this a comment, or did you mean to include this line?

Reply:
commented out, but thanks.